loadflow/loadflow-api/src/test/java/com/powsybl/loadflow/LoadFlowDefaultParametersLoaderMock.java

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.loadflow;
+
+/**
+ * @author Hugo Kulesza {@literal <hugo.kulesza at rte-france.com>}
+ */
+public class LoadFlowDefaultParametersLoaderMock extends AbstractLoadFlowDefaultParametersLoader {
```

Comment:
Couldn't you directly test it by adding `@AutoService(LoadFlowDefaultParametersLoader.class)`?

Reply:
My bad, you need two classes with empty constructor then.

Reply:
and it is bad practice in fact!